### PR TITLE
Tiltas Valley / Rukas Plateau / Constructor's Chapel

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -10444,7 +10444,7 @@ ETC_20150401_010445	{memo X}Grape Ellum
 ETC_20150401_010446	Farm Keposeed
 ETC_20150401_010447	Guards Graveyard
 ETC_20150401_010448	{memo X}2nd Demon Prison
-ETC_20150401_010449	Royal Mausoleum Workers' Lodge
+ETC_20150401_010449	Royal Mausoleum Workers' Chapel
```
Comment (Sunaries) [inline ETC.tsv]: Workers' Chapel -> Constructer's Chapel

File: ETC.tsv [hunk 2]
```diff
@@ -11229,7 +11229,7 @@ ETC_20150428_011230	Dionys
 ETC_20150428_011231	Disgusting Deathweaver
 ETC_20150428_011232	Pyrokinetic Kelberos
 ETC_20150428_011233	Dark Chapparition
-ETC_20150428_011234	Royal Mausoleum Worker's Lodge
+ETC_20150428_011234	Royal Mausoleum Contructor's Chapel
```
Comment (Sunaries) [inline ETC.tsv:11232]: Constructor -> Constructer

File: ETC.tsv [hunk 3]
```diff
@@ -12430,13 +12430,13 @@ ETC_20150714_012431	Call out the name
 ETC_20150714_012432	Recalled a number!{nl}
 ETC_20150714_012433	The seal of Archon has been weakened
 ETC_20150714_012434	~secs remaining
-ETC_20150714_012435	$The purification of the evil energy has been completed!{nl}Light energy has spread throughout the Royal Mausoleum Worker's Lodge!
+ETC_20150714_012435	$The purification of the evil energy has been completed!{nl}Light energy has spread throughout the Royal Mausoleum Contructor's Chapel!
```
Comment (Sunaries) [inline ETC.tsv:12433]: Constructor -> Constructer

File: ETC.tsv [hunk 4]
```diff
@@ -12430,13 +12430,13 @@ ETC_20150714_012431	Call out the name
...
 ETC_20150714_012436	The barrier at the Management Office has been removed!
 ETC_20150714_012437	The altar has already been activated
 ETC_20150714_012438	The light power, that is lethal to the monsters at the chapel area, has been activated
 ETC_20150714_012439	Unable to create Hauberk from invalid coordinates
 ETC_20150714_012440	There are monsters that protect the seal
-ETC_20150714_012441	Usable only in Royal Mausoleum constructor's chapel
+ETC_20150714_012441	Usable only in Royal Mausoleum Contructor's Chapel
```
Comment (Sunaries) [inline ETC.tsv:12439]: Constructor -> Constructer

File: ETC.tsv [hunk 5]
```diff
@@ -14158,19 +14158,19 @@ ETC_20150804_014159	Achieved by being the first to reach rank 6 in CBT3!
...
 ETC_20150804_014165	SP Recovery Rate will increase due to the light effect of the loading place.
 ETC_20150804_014166	Pre Dungeon Defence
 ETC_20150804_014167	Accomplished 4 level of Shield type Equipment Enhancement
 ETC_20150804_014168	Check the epitaph of Great King Zachariel
-ETC_20150804_014169	Until the light device at the Prayer Hall gets activated
-ETC_20150804_014170	The light device at the Prayer Hall at Patama Oratorium has been activated!
-ETC_20150804_014171	The light device at the Prayer Hall has been activated!{nl}The light comes into the Prayer Hall of the constructor of the Mausoleum
+ETC_20150804_014169	Time until the light device for the Prayer Hall gets activated: 
+ETC_20150804_014170	The light device at Fatama Oratorium has been activated!
+ETC_20150804_014171	The light device at Fatama Oratorium has been activated!{nl}The light comes into the Prayer Hall of the Royal Mausoleum Contructor's Chapel
```
Comment (Sunaries) [inline ETC.tsv:14169]: Constructor -> Constructer

File: QUEST.tsv [hunk 6]
```diff
@@ -2007,9 +2007,9 @@ QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to
 QUEST_20150511_002011	$We believe your help is the main factor to our success so that our efforts will not have been in vain.
 QUEST_20150511_002012	$We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to build it. {nl}I believe only this place can answer the question I've had in my mind for a long time.
 QUEST_20150511_002013	$What was he trying to protect with this gigantic tomb? {nl}The answer must be inside, but the bleak atmosphere the inside exudes makes me hesitate to go in.
-QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pillar outside of the tomb.
```
Comment (Sunaries) [inline QUEST.tsv]: tomb -> Royal Mausoleum

File: QUEST.tsv [hunk 7]
```diff
@@ -2007,9 +2007,9 @@ QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to
 QUEST_20150511_002011	$We believe your help is the main factor to our success so that our efforts will not have been in vain.
 QUEST_20150511_002012	$We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to build it. {nl}I believe only this place can answer the question I've had in my mind for a long time.
 QUEST_20150511_002013	$What was he trying to protect with this gigantic tomb? {nl}The answer must be inside, but the bleak atmosphere the inside exudes makes me hesitate to go in.
```
Comment (Sunaries) [inline QUEST.tsv]: tomb -> Royal Mausoleum

File: QUEST.tsv [hunk 8]
```diff
@@ -2007,9 +2007,9 @@ QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to
...
-QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pillar outside of the tomb.
 QUEST_20150511_002015	$With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
```
Comment (Sunaries) [inline QUEST.tsv]: With the help from the young, kind historian, -> With the help of a young, kind historian,?
Ruko Plateau -> Rukas Plateau

File: QUEST.tsv [hunk 9]
```diff
@@ -2019,9 +2019,9 @@ QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous
 QUEST_20150511_002023	$The Complete Research Materials of Varkis
 QUEST_20150511_002024	$The First Research Material{nl}-{nl}We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to build it.{nl}I believe only this place can answer the question I've had in my mind for a long time.
```
Comment (Sunaries) [inline QUEST.tsv]: Zachariel -> Great King Zachariel
Mausoleum -> Royal Mausoleum

File: QUEST.tsv [hunk 10]
```diff
@@ -2019,9 +2019,9 @@ QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous
...
 QUEST_20150511_002025	$What was he trying to protect with this gigantic tomb? {nl}The answer must be inside, but the bleak atmosphere the inside exudes makes me hesitate to go in.
-QUEST_20150511_002026	$The Second Research Material{nl}-{nl}There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002026	$The Second Research Material{nl}-{nl}There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pillar outside of the tomb.
```
Comment (Sunaries) [inline QUEST.tsv]: tomb -> Royal Mausoleum (2x)

File: QUEST.tsv [hunk 11]
```diff
@@ -2019,9 +2019,9 @@ QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous
...
 QUEST_20150511_002027	$The Third Research Material{nl}-{nl}With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
```
Comment (Sunaries) [inline QUEST.tsv:2023]: With the help from the young, kind historian, -> With the help of a young, kind historian,?
Ruko Plateau -> Rukas Plateau

File: QUEST.tsv [hunk 12]
```diff
@@ -2019,9 +2019,9 @@ QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous
...
-QUEST_20150511_002028	$The Fourth Research Material{nl}-{nl}Some poles unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
+QUEST_20150511_002028	$The Fourth Research Material{nl}-{nl}Some pillars unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
```
Comment (Sunaries) [inline QUEST.tsv]: tomb -> Royal Mausoleum (2x)

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -556,12 +556,12 @@ QUEST_LV_0100_20150317_000551	How many things are hiding in this canyon?{nl}More
 QUEST_LV_0100_20150317_000552	According to the old records, the king erased the stonemason's memories using magic to protect the Mausoleum.{nl}I think it's amazing that he erased only those memories.
 QUEST_LV_0100_20150317_000553	Liaison Official Nian
 QUEST_LV_0100_20150317_000554	You saw that, right? That's why I told you not to touch it.{nl}Because of this, we received an order to withdraw from this place.{nl}
-QUEST_LV_0100_20150317_000555	Well, if I am to be involved in this, can you at least do me a favor?{nl}Tell Adelle at the Research Camp to watch out for these kind of stones.
+QUEST_LV_0100_20150317_000555	Well, if I am to be involved in this, can you at least do me a favor?{nl}Tell Adelle at the Excavation Team Camp to watch out for these kind of stones.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:559]: Original says Research Camp. Mistake by IMC?

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -556,12 +556,12 @@ QUEST_LV_0100_20150317_000551	How many things are hiding in this canyon?{nl}More
 QUEST_LV_0100_20150317_000552	According to the old records, the king erased the stonemason's memories using magic to protect the Mausoleum.{nl}I think it's amazing that he erased only those memories.
 QUEST_LV_0100_20150317_000553	Liaison Official Nian
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Liaison Official Nian -> Liaison Officer Nian

File: QUEST_LV_0100.tsv [hunk 15]
```diff
@@ -556,12 +556,12 @@ QUEST_LV_0100_20150317_000551	How many things are hiding in this canyon?{nl}More
 QUEST_LV_0100_20150317_000552	According to the old records, the king erased the stonemason's memories using magic to protect the Mausoleum.{nl}I think it's amazing that he erased only those memories.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Mausoleum -> Royal Mausoleum

File: QUEST_LV_0100.tsv [hunk 16]
```diff
@@ -1459,8 +1459,8 @@ QUEST_LV_0100_20150317_001454	The colleague who I used to work with has disappea
 QUEST_LV_0100_20150317_001455	Where is he. I am scared.
 QUEST_LV_0100_20150317_001456	{memo X}My colleague is dead?{nl}That's nonsense. If I had not tempted him..{nl}
 QUEST_LV_0100_20150317_001457	{memo X}This map is useless now since my colleague is dead.{nl}Please pray that my colleague would go to the Goddess safely.
-QUEST_LV_0100_20150317_001458	Damn. Monsters took my bag. {nl}The bag contained everything that is needed for my journey.. What should I do?{nl}
-QUEST_LV_0100_20150317_001459	The Goddess helped!{nl}Those monsters went down.
+QUEST_LV_0100_20150317_001458	Damn. Monsters took my bag. {nl}The bag contained everything I recorded on my journey.. What should I do?{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Damn -> I'm fucked (slang, but let's not do this :P) /I'm screwed/I'm doomed/I messed up.

File: QUEST_LV_0100.tsv [hunk 17]
```diff
@@ -5954,13 +5954,13 @@ QUEST_LV_0100_20150317_005949	Gustas says there is one seal left but he sensed a
...
 QUEST_LV_0100_20150317_005952	Nian is informing you about the mysterious stone
-QUEST_LV_0100_20150317_005953	Defeat the monster attacking Liaison Officer Nian
+QUEST_LV_0100_20150317_005953	Defeat the monsters attacking Liaison Officer Nian
 QUEST_LV_0100_20150317_005954	Touched the mysterious stone to take a closer look but then monsters appeared! Better defeat the monsters first. 
 QUEST_LV_0100_20150317_005955	Defeated the monsters. Talk to Nian about what happened. 
 QUEST_LV_0100_20150317_005956	Talk to Historian Adelle
 QUEST_LV_0100_20150317_005957	Adelle know it was someone Nian sent. Ask why she is not withdrawing.
 QUEST_LV_0100_20150317_005958	Collect stones with mysterious patterns
-QUEST_LV_0100_20150317_005959	Adelle is also curious about the mysterious stones. She says something is going on and wants to study more sotnes. Collect the stones with mysterious patters from the Hogmas for Adelle.
+QUEST_LV_0100_20150317_005959	Adelle is also curious about the mysterious stones. She says something is going on and wants to study more stones. Collect the stones with mysterious patterns from the Hogmas for Adelle.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Adelle -> Historian Adelle (only in 1st sentence)

File: QUEST_LV_0100.tsv [hunk 18]
```diff
@@ -5954,13 +5954,13 @@ QUEST_LV_0100_20150317_005949	Gustas says there is one seal left but he sensed a
 QUEST_LV_0100_20150317_005950	Released the last seal. Return to Gustas Jonas.
 QUEST_LV_0100_20150317_005951	Talk to Liaison Officer Nian
 QUEST_LV_0100_20150317_005952	Nian is informing you about the mysterious stone
-QUEST_LV_0100_20150317_005953	Defeat the monster attacking Liaison Officer Nian
+QUEST_LV_0100_20150317_005953	Defeat the monsters attacking Liaison Officer Nian
 QUEST_LV_0100_20150317_005954	Touched the mysterious stone to take a closer look but then monsters appeared! Better defeat the monsters first. 
 QUEST_LV_0100_20150317_005955	Defeated the monsters. Talk to Nian about what happened. 
 QUEST_LV_0100_20150317_005956	Talk to Historian Adelle
 QUEST_LV_0100_20150317_005957	Adelle know it was someone Nian sent. Ask why she is not withdrawing.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Adelle -> Historian Adelle

File: QUEST_LV_0100.tsv [hunk 19]
```diff
@@ -6901,19 +6901,19 @@ QUEST_LV_0100_20150511_006896	Tell him that it would be better to take some rest
 QUEST_LV_0100_20150511_006897	$It would be better to stop the attacks destroying the Royal Mausoleum.
 QUEST_LV_0100_20150511_006898	{memo X}Notice/The portal has opened after activating the device!{nl}Check the location of the portal from the map!#7
 QUEST_LV_0100_20150511_006899	The search team told you that Vubbe Warriors are at the felled area, but it would be better to wait for the reinforcement. But, since you should go to the Crystal Mine as quickly as possible, let's first defeat those Vubbe Warriors.
-QUEST_LV_0100_20150511_006900	Retrieve the diary from the lost bag of Varkis
+QUEST_LV_0100_20150511_006900	Retrieve the Research Materials from Varkis' lost bag
 QUEST_LV_0100_20150511_006901	The adventurer Varkis told you that monsters stole the bag and ran away. Retrieve his diary by finding the bag at way below the Shirnas Highland.
 QUEST_LV_0100_20150511_006902	{memo X}Go to the camp of adventurer, Varkis
 QUEST_LV_0100_20150511_006903	{memo X}The gigantic monster was defeated, but Varkis passed away. Returnto his camp with the diary.
-QUEST_LV_0100_20150511_006904	Move to the camp of Varkis
+QUEST_LV_0100_20150511_006904	Go to Varkis' camp
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Go to Varkis' camp -> Go to Camp of Varkis (map name)

File: QUEST_LV_0200.tsv [hunk 20]
```diff
@@ -3034,7 +3034,7 @@ QUEST_LV_0200_20150317_003033	{memo X}You found the divine artifact of the bless
 QUEST_LV_0200_20150317_003034	{memo X}You found the divine artifact of the blessing. Talk with Bishop Aurelius what to do next.
 QUEST_LV_0200_20150317_003035	Look for the divine artifacts of the blessing
 QUEST_LV_0200_20150317_003036	Retrieve the divine artifacts of the blessing by defeating the demons. If you have a trouble retrieving the divine artifacts, call out Bishop Aurelius and ask him.
-QUEST_LV_0200_20150317_003037	The main hall
+QUEST_LV_0200_20150317_003037	The Main Hall
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3037]: I have this in my PR as: Main Chamber

File: QUEST_LV_0100.tsv (hunk 13 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv:559]: Yeah, on this map there's no Research Camp, and the quest icon is at the Excavation Team Camp.

**Comment by terrapin47** [comment]
Thanks :+1:

**Comment by Sunaries** [comment]
:+1:  good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**